WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Dynamic/DynamicDashboardCard.swift

```diff
@@ -39,29 +39,26 @@ struct DynamicDashboardCard: View {
         }
         .padding(.bottom, Length.Padding.single)
         .padding(.horizontal, Length.Padding.double)
-        .fixedSize(horizontal: false, vertical: true)
     }
 
     @ViewBuilder
     var featureImage: some View {
         if let featureImageURL = input.featureImageURL {
             AsyncImage(url: featureImageURL) { phase in
-                Group {
-                    if let image = phase.image {
-                        image
-                            .resizable()
-                            .aspectRatio(contentMode: .fit)
-                    } else {
-                        Color.DS.Background.secondary
-                            .frame(maxWidth: .infinity)
-                            .frame(height: 150)
-                    }
+                switch phase {
+                case .success(let image):
+                    image
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .clipShape(
```

Comment:
I think we want the clip shape to be applied on both the image and the placeholder?